frontend/next.config.js

```diff
@@ -3,6 +3,16 @@ const { default: next } = require('next')
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+
+  webpack (config) {
+    config.module.rules.push({
+      test: /\.svg$/i,
+      issuer: /\.[jt]sx?$/,
+      use: ['@svgr/webpack']
+    })
+
+    return config
+  }
```

Comment:
Why did you install webpack here and what do you need it for?

Reply:
same as Dylan/Jae. allows for svg imgs to be added